lib/Config.js

```diff
@@ -43,6 +43,14 @@ class Config {
             this.port = config.port;
         }
 
+        this.listenOn = [];
+        if (config.listenOn !== undefined) {
+            assert(Array.isArray(config.listenOn)
+                && config.listenOn.every(e => typeof e === 'string'),
```

Comment:
What about an array element of `''`? I don't know why someone would add that, but I suppose it's possible. It looks like the server would then try to listen on `''`.